### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by farshidtz**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+PR description, link to relevant issues
```
Comment (locnnil) [inline .github/pull_request_template.md:1]:
```suggestion
## Description
<!-- PR description, link to relevant issues and summary of changes -->
```

**Review by locnnil — COMMENTED** [review]
The overall structure is great. I've added one minor suggestion.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+## Testing Steps
+<!-- Steps to test the changes. Remove if not relevant -->
+
+## Checklist
```
Comment (jpm-canonical) [inline .github/pull_request_template.md:6]: What is the goal of this checklist? Is it to remind us of what to think about when filing a PR? Or do we want to keep this as record of things that changed? Or do we see this as things that has to be done? Maybe update the heading to reflect what this checklist's purpose is. Ex: "## Did you remember to update these?", "## This PR also affects:"

**Review by jpm-canonical — APPROVED** [review]
It's good enough for now. We can change it over time as we figure out what is required, or as requirements change.

File: .github/pull_request_template.md (hunk 2 above)
Comment (farshidtz) [reply .github/pull_request_template.md]: Changed to comments.

**Review by locnnil — APPROVED** [review]
LGTM!

**Pull request merged**